features/steps/task_execution.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+from behave import given
+
+from bqflow.task.workflow import execute, get_workflow
+
+sys.path.append('features/tasks')
+
+@given('a config file "{config_file}"')
+def step_impl(context, config_file):
+  config = FakeConfiguration()
+  workflow = get_workflow(filepath=f'features/{config_file}')
+  context.log = execute(config, workflow, False, None)
```

Comment:
This type of testing will not support complex tests such as downloading a DV report and writing it to BQ, which is 90% of the work bqflow does.  Can you provide a more robust example, otherwise this test method will be refactored significantly as soon as we try to migrate one of the existing tasks.

Reply:
I was reluctant to make a more complex use case because I wasn't sure if you'd be on board with this kind of framework for integration tests. 

While everything could be loaded through auth when tested locally or through GitHub Actions (with service accounts) my preference was to not prematurely optimize. 

I'm happy to support those tests if they're needed but would prefer to do it in a separate PR where one or two workflows are testable through running `behave`.

Reply:
The tests to cover would be any of the existing tests in the test folder. The existing framework for integration testing uses the logger already in the system and has "features".  Perhaps we stick with that given theres are similar.